crates/astria-conductor/src/executor/tests.rs

```diff
@@ -261,16 +268,21 @@ async fn start_mock() -> MockEnvironment {
     }
 }
 
+fn test_transaction(data: &str) -> RawRollupData {
```

Comment:
Can you prefix this with a verb? Just `make_transaction()` or `make_rollup_data()`